azure-pipelines-release.yml

```diff
@@ -18,26 +18,23 @@ steps:
   displayName: 'Use the .NET 6 SDK'
   inputs:
     packageType: 'sdk'
-    version: '6.0.x'
+    useGlobalJson: true
 
 # Start by restoring all the dependencies. This needs to be its own task.
 - task: DotNetCoreCLI@2
-  displayName: 'Restore nuget dependencies'
+  displayName: 'Restore'
   inputs:
     command: restore
     verbosityRestore: Minimal
-    projects: 'src/**/*.csproj'
+    projects: 'src/dirs.proj'
 
-# Build the entire solution. This will also build all the tests, which
-# isn't strictly necessary...
-- task: VSBuild@1
+# Build source directory
+- task: DotNetCoreCLI@2
```

Comment:
Just for my own curiosity, why'd you decide to go with `DotNetCoreCLI` over `VSBuild`?

Reply:
1) keep it consistent with our other tasks. 2) `VSBuild` can actually have differences with `DotNetCoreCLI`, as they potentially include a different set of targets. By ensuring we use dotnet CLI we won't accidentally take on a dependency to VS-only targets.